File: 20240601070 SE226 LAB1/lab1 - c++/time_logic.cpp
#include <iostream>
using namespace std;

int main() {
int secs{};
cout << "Enter seconds: "<< endl;
cin >> secs;

if (secs <0) {
    cout << "time can't be smaller than 0 seconds";
    return 1;
}

int hours = (secs/3600);
int minutes = ((secs%3600)/60);
int seconds = secs%60;

cout << secs << " seconds is " << hours << " hours, " << minutes << " minutes, and " << seconds << " seconds." << endl;

    return 0;
}

File: 20240601070 SE226 LAB1/lab1 - python/time_logic.py
sec = input("Enter seconds: ")

intsecs = int(sec)

if(intsecs < 0):
    print("time can't be smaller than 0 seconds")
    quit();

hours = intsecs // 3600
minutes = (intsecs % 3600) // 60
seconds = intsecs % 60
print(str(intsecs)+ " seconds is " + str(hours) + " hours, " + str(minutes) + " minutes, and " + str(seconds) + " seconds")

